### ISSUE TITLE
One question

### ISSUE THREAD

**Issue by Zyttty** [tsing90/pytorch_semantic_human_matting#21]
I really appreciate your work. Have you ever tried to replace T-Net's loss function with BCELoss, put alpha_pre and alpha_gt into it?

**Comment by tsing90** [comment]
sorry, I didn't get it, why BCEloss with alpha together?
